factorio

```diff
@@ -442,6 +447,34 @@ case "$1" in
       exit 1
     fi
     ;;
+	
+  new-game)
+    # Stop Service
+    if is_running; then
+      stop_service
+    fi
+
+    if ! as_user "$BINARY --create ${SAVE_NAME}"; then
+      echo "Failed to create new game"
+	  exit 1
+    else
+      echo "New game created"
+    fi
+    ;;
+	
+  save-game)
+    savename=${FACTORIO_PATH}/saves/$2.zip
+
+    # Stop Service
+    if is_running; then
+      stop_service
+    fi
+
+    if ! as_user "cp ${FACTORIO_PATH}/saves/${SAVE_NAME}.zip ${savename}"; then
```

Comment:
Please use ${WRITE_DIR}, se comment on line 466 - this goes for all uses but I won't comment them all ;)